cmd/apps/inletsproclient_app.go

```diff
@@ -0,0 +1,171 @@
+// Copyright (c) arkade author(s) 2020. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package apps
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/alexellis/arkade/pkg/apps"
+	"github.com/alexellis/arkade/pkg/config"
+	"github.com/alexellis/arkade/pkg/k8s"
+	"github.com/alexellis/arkade/pkg/types"
+
+	"github.com/alexellis/arkade/pkg"
+	"github.com/spf13/cobra"
+)
+
+func MakeInstallInletsProClient() *cobra.Command {
+	var inletsProClient = &cobra.Command{
+		Use:          "inlets-pro-client",
+		Short:        "Install inlets-pro client",
```

Comment:
We should mention that it's a TCP client

Reply:
You're right, I've updated the description. Do you think it's reasonable to reflect it in the command as well? like `arkade install inlets-pro-tcp-client`?

Reply:
Perhaps just the name: `inlets-tcp-client`?